.travis.yml

```diff
@@ -23,22 +48,30 @@ addons:
     - g++-4.8
 env:
   matrix:
-  - CXX=g++-4.8
+  - EMBER_TRY_SCENARIO=default
+  - EMBER_TRY_SCENARIO=ember-release
+  - EMBER_TRY_SCENARIO=ember-2-8
   global:
-    secure: m3T8OBefKGIJLK1hogS6WvLXDxGSMX539d6Gazgd9D42Q/iF7ni+58fZM8akQOQMAzNssg82Yv29yS+GSudL68+jKHrzLviZ8gc77FwhLagEHOrxFZWPEdjNwNETkocXX03Rda+Bzo9ZaM+RA2jmJfcxxqu9U9rvjHgqNFyqeVLMrtI8BR0vIlyXEnk9lIroBy8Nw8xTFIQkaB3hJlPjbhnYHQSFlRnn8+gtcdu1ZrlGzm8uLHSacLLlcS0fF42tGeN54eGrgpMhhlOe1Kiu+eCIxnq96uoRWwKSfw+8WqidH/0CTcTaYirIXNdxPjUU3AyHeItmOG7TidAbrhfibu8S57OvVy5dL2cxixx8NRZBwmra0HEA0qEdP381K4Re6rJhJNL93B24SWw0UL7X2KmoFBjiPFYoom8VUm9v6rIe0YMtGYp9a6wBzI+Wx5hpdRQrWG9Rj2H6Sbnrfhl2SfMQwCPyHMpIxLfNAwsV4V6smX0w3MqmLHnzWdLj14k0ETOFVjOpcFxVeza/jo7XLW5FdpEgAxSZXDTI8qVMl2ViqeezTo9EKCHBylOEKIT3/usXbWpN5qAQWR7M/qyua31tWDrgsiNFpWncJQaFcRDF30bvULznR+GILEEk9ZwiBIgeGucsAkDEzQiPY62I8tRx7B26OK2X/sBPW09qNIk=
-before_deploy:
-- pr-bumper bump
+    - CXX=g++-4.8
+    - secure: m3T8OBefKGIJLK1hogS6WvLXDxGSMX539d6Gazgd9D42Q/iF7ni+58fZM8akQOQMAzNssg82Yv29yS+GSudL68+jKHrzLviZ8gc77FwhLagEHOrxFZWPEdjNwNETkocXX03Rda+Bzo9ZaM+RA2jmJfcxxqu9U9rvjHgqNFyqeVLMrtI8BR0vIlyXEnk9lIroBy8Nw8xTFIQkaB3hJlPjbhnYHQSFlRnn8+gtcdu1ZrlGzm8uLHSacLLlcS0fF42tGeN54eGrgpMhhlOe1Kiu+eCIxnq96uoRWwKSfw+8WqidH/0CTcTaYirIXNdxPjUU3AyHeItmOG7TidAbrhfibu8S57OvVy5dL2cxixx8NRZBwmra0HEA0qEdP381K4Re6rJhJNL93B24SWw0UL7X2KmoFBjiPFYoom8VUm9v6rIe0YMtGYp9a6wBzI+Wx5hpdRQrWG9Rj2H6Sbnrfhl2SfMQwCPyHMpIxLfNAwsV4V6smX0w3MqmLHnzWdLj14k0ETOFVjOpcFxVeza/jo7XLW5FdpEgAxSZXDTI8qVMl2ViqeezTo9EKCHBylOEKIT3/usXbWpN5qAQWR7M/qyua31tWDrgsiNFpWncJQaFcRDF30bvULznR+GILEEk9ZwiBIgeGucsAkDEzQiPY62I8tRx7B26OK2X/sBPW09qNIk=
+
+matrix:
+  fast_finish: true
+  allow_failures:
+  - env: EMBER_TRY_SCENARIO=ember-release
```

Comment:
Most likely we will need to allow failures on `stable` as well for the time being.

`- node_js: 'stable'`